DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -198,6 +207,12 @@ class CaloTower : public reco::LeafCandidate {
    GlobalPoint emPosition_;
    GlobalPoint hadPosition_;
 
+  //hcal limits in CT continuous ieta (from CaloTowerTopology)
+  int firstHBRing_, lastHBRing_;
```

Comment:
I will set them as transients. They are necessary to make the energyinHB/HE/HF/HO() functions work now that the ieta bounds for each HCAL subdetector are not hard-coded.